doc/source/dev/development_environment.rst

```diff
@@ -211,10 +211,20 @@ Debugging
 ---------
 
 Another frequently asked question is "How do I debug C code inside NumPy?".
-The easiest way to do this is to first write a Python script that invokes the C
-code whose execution you want to debug. For instance ``mytest.py``::
+First you must obtain the correct version of gdb (often the default on
+Linux), which requires an embedded Python. To see which version of Python
```

Comment:
Comparing this to the deleted text (in what used to be lines 231-232 in the diff) it seems that the meaning has changed a bit. For example, the parenthetical statement below was referring to the fact that the Python extensions for gdb are often installed by default on Linux, whereas it's placement here indicates that "the correct version" of gdb is the default on Linux.

I like the addition of the example below explaining how to test the embedded Python from within the gdb session, but I think that the wording here needs a little work to recapture the original meaning of the deleted sentence.

Reply:
yes, the meaning here is unclear, added more accurate version description.